client/app/bundles/course/admin/admin.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(() => {
+  $('input.self-directed-learning-checkbox:checkbox').change(
```

Comment:
Will adopt `advance-start-at` to CSS classes, it's always good to keep them consistent 👍